contrib/mesos/pkg/minion/tasks/task.go

```diff
@@ -218,10 +219,15 @@ func notStartedTask(t *Task) taskStateFn {
 
 	// create command
 	cmd := exec.Command(t.bin, t.args...)
-	if _, err := cmd.StdoutPipe(); err != nil {
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
 		t.tryError(fmt.Errorf("error getting stdout of %v: %v", t.name, err))
 		return taskShouldRestart
 	}
+	go func() {
+		defer stdout.Close()
+		io.Copy(ioutil.Discard, stdout) // TODO(jdef) we might want to save this at some point
```

Comment:
glog pushes everything to stderr. stdout is really only used by the
detailed connection-spying stuff in mesos-go (which is why the mgo upgrade
broke when stdout blocked and verbosity is jacked up). I'm happy to do
address logging this to a file in a separate PR. This PR is simply to stop
the breakage.

On Fri, Oct 16, 2015 at 1:41 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/minion/tasks/task.go
> <https://github.com/kubernetes/kubernetes/pull/15735#discussion_r42210150>
> :
>
> >  		t.tryError(fmt.Errorf("error getting stdout of %v: %v", t.name, err))
> >  		return taskShouldRestart
> >  	}
> > +	go func() {
> > +		defer stdout.Close()
> > +		io.Copy(ioutil.Discard, stdout) // TODO(jdef) we might want to save this at some point
>
> Should't we log this as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15735/files#r42210150>.
>